from system.application.exceptions.repository_exceptions import DataRepositoryExeption


class PostgreSQLError(DataRepositoryExeption):
    pass
